sys/net/network_layer/ng_sixlowpan/ctx/ng_sixlowpan_ctx.c

```diff
@@ -36,7 +36,7 @@ static char ipv6str[NG_IPV6_ADDR_MAX_STR_LEN];
 static inline bool _still_valid(unsigned int id)
 {
     _update_lifetime(id);
-    return (_ctxs[id].ltime > 0);
+    return (_ctxs[id].prefix_len > 0);
```

Comment:
Why? `prefix_len` becomes the signifier for empty ctx in this PR, so how is this confusing?